components/footer/footer.njk

```diff
@@ -1 +1,73 @@
-<footer></footer>
+<footer class="footer">
+  <div class="footer__top">
+    <div class="footer__top__container">
+      <h4 id="footer-partners" class="footer__heading">{{ footer.partners_title }}</h4>
+      <ul class="footer-partners__list inline-list" aria-labelledby="footer-partners">
+        {% for partner in footer.partners %}
+          {% if partner.link %}
+            {# here we use aria-label for screen readers, and hide the image itself from them #}
+            <a class="footer__partner-logo"
+              href="{{ partner.link }}"
+              target="_blank"
+              rel="noopener noreferrer"
+              aria-label="{{ partner.title }} (opens in a new tab)">
+                <img src="{{ partner.image_url }}" alt="{{ partner.title }}" role="presentation" aria-hidden="true">
```

Comment:
Why not leave alt tag empty at all?

Reply:
And what about title tag for the link?

The whole thing confuses me. 😟

Reply:
yeah, could leave it empty, it shouldn't matter as long as `role` is `presentation`. but yeah, I should use `title` instead of `aria-label`